.claude/scripts/portfolio-surveyor.test.sh

```
@@ -1251,6 +1251,28 @@ grep -Fq 'shared tip alone is enough' "${surveyor}" ||
 grep -Fq '`.claude/scripts/agent-claim.sh retire <issue> "$claim_sha" --repo-dir <selected-repo-path>`' "${cursor_loader}" ||
   fail "Cursor loader retirement does not invoke the repository-qualified helper"
 
+# Same-repo PR-body claim filter (#2250) — echoed at every site that tells a run how to decide
+# "no open PR". `-R` scopes the PR *list* to this repo, but a body can still name a foreign
+# `owner/repo#<issue>`, which is not a claim on this repo's issue; counting it hides the oldest
+# actionable issue behind an unrelated PR.
+grep -Fq 'foreign `owner/repo#<issue>`' "${constitution}" ||
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)